change/@internal-react-composites-2bfa76c6-79a1-4af1-ba29-dfa1a0e23c01.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "minor",
+  "comment": "Add Custom button injection for CallWithChat and Call Composite",
```

Comment:
Technically this already exists in beta for call composite. Can you mention the change we're making here to it

Reply:
Good point! Revised to:
Rework how Custom buttons are injected in Call Composite and add Custom Button Injection into CallWithChat Composite

A bit long but explains the changes.